app/views/notifications/index.html.erb

```diff
@@ -0,0 +1,16 @@
+<% provide :title, t(".notifi_index_title") %>
+<h1><%= t ".heading" %></h1>
+
+<dir class="container" >
+	<div class="row" >
+	  <div class="col-md-6 col-md-offset-3" >
+			<ul class="notification-index-list" >
+		  	<%= render @notifications %>
+			</ul>
+	  </div>
+	</div>
+</dir>
```

Comment:
![Uploading image.png…]()

Reply:
sorry i cant see this picture. May u resend it